### DISCUSSION TITLE
Physics in SK

### DISCUSSION THREAD

**Discussion by laultman** [StereoKit/StereoKit#854] [Q&A]
@maluoi I read the note in the source about Solid.cs and the reference to [bepu ](https://github.com/bepu/bepuphysics2) It said on the ReadMe page that it supports .NET 6. Do you have any knowledge whether this lib will work in HoloLens 2. Just checking before jumping on it?

**Comment by maluoi** [comment]
Bepu [does not target netstandard](https://github.com/bepu/bepuphysics2/issues/194) or UWP, so it does look like it won't work on HoloLens! Sounds like there could be a fork that might be netstandard, but that likely won't be hosted on NuGet.
